src/worker/ingestion/process-event.ts

```diff
@@ -189,6 +210,9 @@ export class EventsProcessor {
         distinctId: string,
         teamId: number
     ): Promise<void> {
+        if (isDistinctIdIllegal(distinctId)) {
+            return
```

Comment:
Can we also add `statsd` instrumentation (sending `distinctId` too) here to have at least an internal metric of the scale of the issue?